core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts

```diff
@@ -470,7 +471,18 @@ export class ServiceLogsComponent implements OnInit {
 		$(".pagination.justify-content-center li:nth-child(2)")[0].click();
 		this.callLogsFunc();
 	}
-
+	assetsModify(){
```

Comment:
Can we name this method to be more indicative of the purpose?

Reply:
done